### PR TITLE
Tickets/dm 29601

### PR DESCRIPTION
**Pull request by jeffcarlin**
Rendered version is [here](https://dmtr-311.lsst.io/v/DM-29601/index.html).

### CODE REVIEW

File: DMTR-311.tex [hunk 1]
```diff
@@ -15,17 +24,16 @@
...
 
 \setDocRef{DMTR-311}
 \setDocUpstreamLocation{\url{https://github.com/lsst-dm/DMTR-311}}
 \date{\vcsDate}
 % \setDocCurator{The Curator of this Document}
 
 \setDocAbstract{%
-Add abstract text.
+This brief report describes measurements of data quality metrics that were carried out for release v22.0.0 of the LSST Science Pipelines.
+The report for the previous version can be found in \citedsp{DMTR-281}.
```
Comment (leannep) [inline DMTR-311.tex:36]: On the landing page, this reference does not render

File: DMTR-311.tex [hunk 2]
```diff
@@ -40,8 +48,1000 @@
 
 \maketitle
 
-% ADD CONTENT HERE
-% You can also use the \input command to include several content files.
+In this Report, we characterize the performance of the Rubin Observatory Science Pipelines Version 22.0.0. We illustrate the performance via metrics that are measured on the HSC-RC2 dataset. RC2 consists of 3 tracts of data taken from the HSC-SSP survey, and selected to provide a means of testing various ``pathological'' cases (e.g., difficult astrometric solutions, seeing too small to provide a well-sampled PSF, difficult fields for deblending, and large galaxies, among others). These three tracts each contain between 112--149 visits split between the HSC-G, HSC-R, HSC-I, HSC-Z, and HSC-Y (\emph{grizy}) filters. 
+
```
Comment (leannep) [inline DMTR-311.tex:52]: seeing is usually characterized as   excellent/bad/good/poor not  small

File: DMTR-311.tex [hunk 3]
```diff
@@ -40,8 +48,1000 @@
...
+Beginning in March 2021, the metrics calculation package \texttt{validate\_drp} was deprecated and replaced with \texttt{faro}, a new package built to use the Gen3 butler, and that is part of the standard science pipeline builds. Henceforth \texttt{faro} will be the tool we use to measure data quality metrics for Rubin Observatory. Before adopting \texttt{faro}, detailed verification was performed to confirm that, given the same input datasets and configuration parameters, \texttt{faro} and \texttt{validate\_drp} produce identical results. Once this parity was demonstrated, \texttt{faro} was adopted and will now evolve (i.e., it will no longer be expected to produce identical results to those from \texttt{validate\_drp}).
+
```
Comment (leannep) [inline DMTR-311.tex:56]: Add a statement that we ported many of the algorithms from validate_drp and have improved on them in faro and also added new metrics.   (I don't want it to seem like we threw away the scientifc work of validate_drp)

**Review by leannep — CHANGES_REQUESTED** [review]
Looks good. Do you have any explanation for the minor difference seen in teh astrometric metrics? Could it be due to the previous non-deterministic nature of validte_drp?

**Comment by jeffcarlin** [comment]
> Looks good. Do you have any explanation for the minor difference seen in teh astrometric metrics? Could it be due to the previous non-deterministic nature of validte_drp?

I don't necessarily have any explanation -- the differences are all small, and given how many changes we've made in the calculation, they are surprisingly consistent.

File: DMTR-311.tex (hunk 1 above)
Comment (jeffcarlin) [reply DMTR-311.tex:36]: Right - I have to update the lsst-tex bibliography to add that...

**Pull request merged**